meshtastic/node_runtime/seturl_runtime.py

```
@@ -0,0 +1,928 @@
+"""setURL transaction runtime owners and planning/execution boundaries."""
+
+import base64
+import binascii
+import logging
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Callable, NoReturn
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)